server/CMakeLists.txt

```diff
@@ -16,7 +16,13 @@ include_directories(${boost_include_dirs})
 
 # here we choose who provides us with the FFT lib
 if (APPLE)
-	add_definitions("-DSC_FFT_VDSP")
+	if (FFT_GREEN)
+		message(STATUS "Using green fft")
+		set(FFT_GREEN 1)
```

Comment:
why is this necessary? we already got here with FFT_GREEN equal to a CMake-truthy value

Reply:
You're right, this is a copy-paste from the code below. I'll remove that.